Cargo.toml

```diff
@@ -6,9 +6,9 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 # revm
-revm = { git = "https://github.com/scroll-tech/revm", default-features = false, features = ["enable_eip7702", "enable_eip7623"] }
-revm-primitives = { git = "https://github.com/scroll-tech/revm", default-features = false }
-revm-inspector = { git = "https://github.com/scroll-tech/revm", default-features = false }
+revm = { git = "https://github.com/scroll-tech/revm", default-features = false, branch = "feat/v88", features = ["enable_eip7702", "enable_eip7623"] }
```

Comment:
This is the current diff, correct? https://github.com/scroll-tech/revm/compare/feat/v88..bluealloy:revm:v88